tests/telemetry/test_telemetry.py

```diff
@@ -102,24 +144,18 @@ def test_env_var_takes_precedence(monkeypatch,
                                         stats_enabled: {yaml_value}
                                         """)
 
-    assert telemetry.check_stats_enabled() is expected_first
+    assert telemetry.check_telemetry_enabled() is expected_first
 
     monkeypatch.setenv('PLOOMBER_STATS_ENABLED', env_value, prepend=False)
 
-    assert telemetry.check_stats_enabled() is expected_second
+    assert telemetry.check_telemetry_enabled() is expected_second
 
 
 def test_first_usage(monkeypatch, tmp_directory):
     monkeypatch.setattr(telemetry, 'DEFAULT_HOME_DIR', '.')
 
-    stats = Path('stats')
-    stats.mkdir()
-
-    # This isn't a first time usage since the config file doesn't exist yet.
-    assert not telemetry.check_first_time_usage()
-    (stats / 'config.yaml').write_text("stats_enabled: True")
-
     assert telemetry.check_first_time_usage()
+    assert not telemetry.check_first_time_usage()
 
```

Comment:
How does this pass, now that we don't set the conf?

Reply:
I added a one-off flag to the config to replicate this functionality

Reply:
Cool just make sure it works via the dashboard